packages/keystone/KSv5v6/v5/prepareKeystone.js

```diff
@@ -151,6 +151,7 @@ function prepareKeystone ({ onConnect, extendKeystoneConfig, extendExpressApp, s
                 // NOTE: if key not found returns 0, else 1.
                 await redisClient.expire(sessKey, SERVICE_USER_SESSION_TTL_IN_SEC)
             },
+            ...get(authStrategyOpts, 'hooks', {}),
```

Comment:
Should you compose hooks, like in plugins?

Reply:
Why not
Let's compose default hooks, written in prepareKeystone and exported from app hooks